tests/login.rs

```diff
@@ -0,0 +1,28 @@
+#![cfg(feature = "http_client")]
+
+use chilloutvr::{
+	api_client::{ApiClient, ApiError, ApiConfiguration},
+	query::{LoginCredentials, AuthType},
+};
+
+mod common;
+
+#[tokio::test]
+#[ignore]
+async fn login() -> Result<(), ApiError> {
+	let config = ApiConfiguration::new(common::USER_AGENT.to_owned());
+	let client = chilloutvr::api_client::UnauthenticatedCVR::new(config).expect("Failed to create unauthenticated client.");
+
+	let query = AuthType::LoginProfile(LoginCredentials {
+		email: "email@Address".to_owned(),
+		password: "pa$$word".to_owned(),
+	});
+
+	let results = client.query(query).await?.data;
+
+	dbg!(&results);
+
+	assert!(!results.access_key.is_empty());
+
+	Ok(())
+}
```

Comment:
Testing for failures seems a bit weird, but I guess it does at least validate that the response gets parsed properly. Could change the test name to `login_invalid` or similar, and I'd use a more clearer testing email like `cvr_rs@example.org`, though both are just nitpicks :stuck_out_tongue:

Probs would be neater to have a test that uses actual (non-committed due to secrets) credentials file to test with an actual login, but dunno about if that's a good idea either..

Reply:
I actually tested using real credentials in there but obviously I would not commit that. reading from a file would make more sense but also storing username + pass in a plaintext file for testing purposes seems kind of weird

also the testing email is the same one as in the README file